src/locale/misc/views/en.yml

```diff
@@ -146,6 +146,9 @@ suggested:
   created: Created by {name}
   sectionTitle: Suggested
 surveyPreview:
+  matches:
+    notSelected: Not selected
+    selected: Selected
```

Comment:
Not sure I agree that these strings belong here, because the owner of the string is not the survey preview component, but the survey option cell. But that's one of _many_ ways in which localized strings lack pattern in this project, so it's not something we need to fix now.